src/components/Section/Foreign/Business/Employment.jsx

```diff
@@ -42,14 +42,15 @@ export default class Employment extends SubsectionElement {
   }
 
   summary(item, index) {
-    const obj = item || {}
-    const date = DateSummary(item.Date)
-    const job = `${(obj.Description || {}).value || ''}`.trim() || ''
+    const o = (item || {}).Item || {}
+    const date = DateSummary(o.Dates)
+    const name = NameSummary(o.Name)
+    const type = i18n.t('foreign.business.employment.collection.summary.item')
```

Comment:
Should `type` be used in the object below?

Reply:
Thanks for that `const type` was not needed for either of these updates 👍